addons/pos_restaurant/static/src/app/screens/floor_screen/floor_screen.js

```diff
@@ -356,15 +354,6 @@ export class FloorScreen extends Component {
             this.state.floorWidth = `${positionH}px`;
         }
     }
-    async onWillStart() {
-        this.pos.searchProductWord = "";
-        const table = this.pos.selectedTable;
-        if (table) {
-            await this.pos.unsetTable();
-        }
-        // Set order to null when reaching the floor screen.
-        this.pos.setOrder(null);
-    }
```

Comment:
keep this method, maybe rename it with `resetTable`, then call it in `onMounted`.